app/assets/stylesheets/application.scss

```diff
@@ -8,4 +8,8 @@ $hmrc-assets-path: "/emcs/account/assets/lib/hmrc-frontend/hmrc/assets/";
 
 .white-space-nowrap {
   white-space: nowrap;
+}
+
+.emcs-tfe-frontend-back-link {
+    margin-top: 30px;
```

Comment:
Probably better to use the existing `!important` CSS classes that you can use for alignment within the GovUk stylesheets.

https://design-system.service.gov.uk/styles/spacing/#overriding-spacing

This is a better option as the spacing resizes dynamically based on the screen size which is important for smaller devices such as mobiles.